examples/remarketing/add_customer_match_user_list.py

```diff
@@ -571,11 +630,19 @@ def normalize_and_hash(s, remove_all_whitespace):
             "not specified, this example will create a new job."
         ),
     )
+    # Need to instantiate a GoogleAdsClient to access enums, but this is only
+    # used for help text, so it's fine if it fails.
+    try:
+        consent_status_enum_names = [
+            e.name for e in GoogleAdsClient.load_from_storage(version="v19").enums.ConsentStatusEnum
+        ]
+    except Exception:
+        consent_status_enum_names = ["UNSPECIFIED", "UNKNOWN", "GRANTED", "DENIED"]
```

Comment:
My understanding is that this would be used for the help text in the case of an exception. If my understanding is correct about this being help text, then only GRANTED and DENIED are recommended values to set for consent. UNKNOWN is output-only, so you'll get an error if setting to UNKNOWN. You can use UNSPECIFIED, but it's like setting the value to empty, and it has no effect.

Reply:
I updated so that `UNSPECIFIED` and `UKNOWN` are filtered out when the options are listed.